### PR TITLE
Testing cdk stack

### PR DESCRIPTION
**Pull request by tehzwen**

### CODE REVIEW

File: __tests__/cdk/custom-resources-base-stack.test.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          `),
+          timeout: 5,
+          environment: {},
+          runtime: Runtime.PYTHON_3_8
+        }
+      })
+    });
+
+    it('Succeeds in building custom handler', () => {
+      console.log(nestedStack);
```
Comment (jessedoyle) [inline __tests__/cdk/custom-resources-base-stack.test.ts]: Do we need this console.log?

**Review by jessedoyle — CHANGES_REQUESTED** [review]
Looking good! I think there may be a `console.log` used for debugging left over.

File: __tests__/cdk/custom-resources-base-stack.test.ts (hunk 1 above)
Comment (tehzwen) [reply __tests__/cdk/custom-resources-base-stack.test.ts]: Good catch! I've removed it 👍🏻

**Review by jessedoyle — APPROVED** [review]
Good work, nice tests!

**Pull request merged**